lua/neoclip/handlers.lua

```diff
@@ -5,16 +5,26 @@ local storage = require('neoclip.storage')
 local settings = require('neoclip.settings').get()
 
 local function should_add(event)
+    if type(settings.length_limit) == "number" then
+        local length = 0
+        for _,line in ipairs(event.regcontents) do
+            length = length + #line
+        end
+        if length > settings.length_limit then
+            return false
+        end
```

Comment:
```suggestion
        if #table.concat(event.regcontents) > settings.length_limit then
            return false
        end
```
I think you could do this instead

Reply:
This is much simpler, but I'm wondering whether the performance will be worse :thinking:

Reply:
Hmm, you think so? Might be interesting to try

Reply:
As `table.concat(event.regcontents)` generates a new string, I think the performance would be worse. I don't think the difference would be noticeable, but the current implementation is also not too complicated, so I think it's OK.

Reply:
Looks good now :)